.pre-commit-config.yaml

```
@@ -26,10 +26,15 @@ repos:
                 files: python/.*\.(py|pyx|pxd)$
                 types: [file]
       - repo: https://github.com/pre-commit/mirrors-mypy
-        rev: 'v0.782'
+        rev: 'v0.971'
         hooks:
               - id: mypy
-                args: ["--config-file=setup.cfg", "python/cudf/cudf", "python/dask_cudf/dask_cudf", "python/custreamz/custreamz", "python/cudf_kafka/cudf_kafka"]
+                additional_dependencies: [types-cachetools==5.2.1]
```

I'd leave this unpinned. It's going to go out of date too quickly, and we left `cachetools` unpinned in the corresponding conda environment.
```suggestion
                additional_dependencies: [types-cachetools]
```